src/models/behaviors.js

```diff
@@ -47,15 +49,17 @@ function missingRequiredFields (obj) {
 
 function addWaitErrors (config, errors) {
     if (!ofType(config.wait, 'number', 'string') || (typeof config.wait === 'number' && config.wait < 0)) {
-        errors.push(exceptions.ValidationError('"wait" value must be an integer greater than or equal to 0',
-            { source: config }));
+        errors.push(exceptions.ValidationError('"wait" value must be an integer greater than or equal to 0', {
```

Comment:
If you can help it, try to avoid these whitespace changes. It's not too big a deal, but it does make reviewing the diffs a little harder. If you review your diffs before committing, you'll probably find blobs like this that your editor unhelpfully changed without you even realizing it and you can undo those there.